### PR TITLE
User/morgilad/integration tests

### PR DESCRIPTION
**Pull request by morgilad**
Basic integration tests.

- added dev script to run monaco-kusto using vite
- added test script to run integration tests locally during development
- added a Jacques Cousteau in Monaco generated AI favicon for local env 

What is missing:
- I need set headless mode for CI and fast testing
- would like to trigger tests on file change
- need to decide how to split tests logic. In the meantime it is too early

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,6 @@
     "devDependencies": {
         "@types/node": "^20.6.3",
         "@types/wait-on": "^5.3.1",
-        "playwright": "^1.32.2",
```
Comment (maxburs) [inline package.json:22]: This still needs to be here, it's used by /scripts/test-samples.ts which runs in CI

File: package/package.json
Comment (maxburs) [inline package/package.json:1]: It looks like this whole file got reformatted to use 2 spaces? I'd expect formatting to use the root /.prettierrc.json. You should be able to configure your editor to format .json using prettier.

**Comment by maxburs** [comment]
If we're going to add app-like things for testing or dev scenarios, they'll be more maintainable if we make them different packages, like the samples are.

Really, it would make a lot more sense to use a separate app for testing too, rather than `browser-sync`. I always meant to make this change, but never got around to it.

Does this would mean local development and testing would better match would people will see when they consume the package.

File: package/package.json
Comment (morgilad) [reply package/package.json:1]: Hi yes. I am checking it. Thanks.

**Review by sagivf — DISMISSED** [review]
Goofd stuff

**Pull request merged**